src/cr/cube/slices.py

```diff
@@ -528,7 +495,9 @@ def name(self):
     @lazyproperty
     def ndim(self):
         """int count of dimensions for this strand, unconditionally 1."""
-        return 1
+
+        # The value of the `ndim` on the `self._stipe` is also unconditionally 1
+        return self._stripe.ndim
```

Comment:
If this is always 1, it's better to say so here so a reader doesn't need to refer to a collaborator to get that news. If the stripe doesn't need `.ndim` for its own purposes, we should remove it there, but it's no violation of principle for us to make this a constant value and it improves readability.